src/components/Container.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import "./Container_style.css";
```

Comment:
Naming convention. No esta mal pero podemos organizar de mejor manera los componentes en `src/components`.

Este post te puede ayudar https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76